docs/release_notes/v0.11.0.md

```diff
@@ -0,0 +1,489 @@
+
+If you previously installed Dapr using __Helm 2.X__:
+
+```bash
+helm del --purge dapr
+```
+
+If you previously installed Dapr using __Helm 3.X__:
+
+```bash
+helm uninstall dapr -n dapr-system
+```
+
+Update the Dapr repo:
+
+```bash
+helm repo update
+```
+
+If you installed Dapr with Helm to a namespace other than `dapr-system`, modify the uninstall command above to account for that.
+
+You can now follow [these](https://github.com/dapr/docs/blob/master/getting-started/environment-setup.md#using-helm-advanced) instructions on how to install Dapr using __Helm 3__.
+
+Alternatively, you can use the CLI:
+
+```
+dapr init --kubernetes
+```
+
+#### Post installation
+
+Verify the control plane pods are running and are healthy:
+
+```
+dapr status -k
+```
+
+After Dapr 0.11.0 has been installed, perform a rolling restart for your deployments to pick up the new version of the sidecar.
+This can be done with:
+
+```
+kubectl rollout restart deploy/<deployment-name>
+```
+
+## Breaking Changes
+
+### Dapr Runtime
+
+#### New ClusterRole for dapr-operator
+
+Removed Cluster Admin permissions and now the dapr-operator has more fine grained [permissions](https://github.com/dapr/dapr/issues/1997). 
+
+#### Serialization for gRPC State Transaction APIs have changed 
+
+Before v0.11.0, on calling the State Transaction API via gRPC, the runtime was converting byte arrays to string and calling the respective component to handle the transaction. 
+
+```go
+Value:    string(req.Value),
+```
+
+Now the runtime no longer coverts the byte array to string, and the individual component needs to handle the serialization properly. 
+
+```
+Value:    req.Value,
+```
+
+Read more [here](https://github.com/dapr/dapr/issues/2016).
+
+
+### Components Contrib 
+
+#### Serialization for State Transaction APIs have to be handled in component 
+
+Before v0.11.0, some of the components (eg: redis, postgres) did not handle State Transaction Upsert request value serialization similar to how it was handled in the corresponding `Set` API path. Thus there was a change made in the runtime to convert byte array request values into string which was then passed on to be handled by the component. This caused an issue where properly serialized values were serialized again.
+
+```go
+// prior to 0.11.0 in state/redis/redis.go
+b, _ := r.json.Marshal(req.Value)
+```
+
+For v0.11.0, the individual component needs to handle the serialization of the values properly, an example for the redis state transaction upsert operation is:
+
+
+```go
+//defined in state/utils/utils.go
+
+func Marshal(val interface{}, marshaler func(interface{}) ([]byte, error)) ([]byte, error) {
+	var err error = nil
+	bt, ok := val.([]byte)
+	if !ok {
+		bt, err = marshaler(val)
+	}
+
+	return bt, err
+}
+
+// in state/redis/redis.go
+bt, _ := utils.Marshal(req.Value, r.json.Marshal)
+```
+
+Read more [here](https://github.com/dapr/dapr/issues/2016).
+
+### Java SDK 
```

Comment:
Is this a breaking change or an addition? /cc @artursouza

Reply:
Yes this is a breaking change.